main.js

```diff
@@ -47,18 +47,45 @@ function validator(emailAcquire) {
   return false;
 }
 
+// Portfolio: preserve data in the browser
+const nameData = document.querySelector('#name');
+const messageData = document.querySelector('#message');
+const emptyObject = {};
+
+function storeData() {
+  emptyObject.namee = nameData.value;
+  emptyObject.email = emailE.value;
+  emptyObject.message = messageData.value;
+  localStorage.setItem('object', JSON.stringify(emptyObject));
+}
+
 // Message to display Depending on Email validation status
 form.addEventListener('submit', (e) => {
   if (validator(emailE.value)) {
     error.textContent = '';
+    storeData();
   } else {
+    storeData();
```

Comment:
- [ ] Thank you for you clear explanations :medal_sports: :medal_sports:
However,  **As suggested by the previous code reviewer,** I think it would be better to always save data to the local storage when the user changes the content of any input field, and to make this, It would be better to change the used ```eventListener``` and make it ```'change'``` instead of ```submit```.  :joy: 
Your project only stores data when user **submits** and it should be on every **change** made, Kindly fix this issue for the best user experience :pray: